scripts/clone-folder.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+FOLDER=$1
+BASE=$(pwd)
+
+for folder in $FOLDER/*; do
+  [ -d "$folder" ] || continue # only directories
+  cd $BASE
+
+  NAME=$(cat $folder/package.json | jq -r '.name')
+  CLONE_DIR="__${NAME}__clone__"
+
+  git clone https://$GITHUB_TOKEN@github.com/dschau/$NAME.git $CLONE_DIR
```

Comment:
Before this is merged, we will need to create a GITHUB_TOKEN environment variable in CircleCI which has `repo` permission.